crates/sdk/Cargo.toml

```diff
@@ -24,8 +24,8 @@ rustls = ["reqwest/rustls"]
 ndc-test = ["dep:ndc-test"]
 
 [dependencies]
-ndc-models = { git = "http://github.com/hasura/ndc-spec.git", tag = "v0.1.3" }
-ndc-test = { git = "http://github.com/hasura/ndc-spec.git", tag = "v0.1.3", optional = true }
+ndc-models = { git = "http://github.com/hasura/ndc-spec.git", branch = "samirtalwar/bump" }
+ndc-test = { git = "http://github.com/hasura/ndc-spec.git", branch = "samirtalwar/bump", optional = true }
```

Comment:
This needs to change (and `Cargo.lock` needs to be updated) before we can merge this changeset.